project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/sample/SamplePreviewJpaRepository.java

```diff
@@ -149,19 +149,23 @@ public static Specification<SamplePreview> conditionContains(String filter) {
       };
     }
 
+    private static Specification<SamplePreview> ontologyColumnContains(String col, String filter) {
+      return (root, query, builder) -> {
+        Expression<String> expression = root.get(col).as(String.class);
+        return builder.like(expression, "%label%"+filter+"%name%");
+      };
+    }
```

Comment:
While this is an awesome quick fix, i fear that this hardcoded search gets lost when we want to adapt it later. Therefore i looked into how this could be achieved and would suggest the following:

```suggestion
    private static Specification<SamplePreview> ontologyColumnContains(String col, String filter) {
      return (root, query, builder) -> {
        /*We're currently only interested in the label within the json file*/
        Expression<String> function =  builder.function("JSON_EXTRACT", String.class, root.get(col), builder.literal("$.label"));
        return builder.like(function,
            "%" + filter + "%");
    }
```